mParticle-Singular/MPKitSingular.m

```diff
@@ -123,6 +130,11 @@ - (MPKitExecStatus *)setDeviceToken:(NSData *)deviceToken {
 - (nonnull MPKitExecStatus *)continueUserActivity:(nonnull NSUserActivity *)userActivity restorationHandler:(void(^ _Nonnull)(NSArray * _Nullable restorableObjects))restorationHandler {
     
     if([userActivity.activityType isEqualToString:NSUserActivityTypeBrowsingWeb]){
+        ddlLink = nil;
+        [Singular registerDeferredDeepLinkHandler:^(NSString *deeplink) {
+            ddlLink = [[NSDictionary alloc] initWithObjectsAndKeys:deeplink,SINGULAR_DEEPLINK_KEY, nil];
```

Comment:
similar to during kit initialization, you should be invoking the completion handler property here.